### ISSUE TITLE
2018.7.18 Kaggle-house-price

### ISSUE THREAD

**Issue by wanxinzzz** [IEC-lab/Summer2018#5]
### 肖航
`def get_net():`
`    net = nn.Sequential()`
`    net.add(nn.Dense(1024, activation='relu'))`
`    net.add(nn.Dropout(0.5))`
`    net.add(nn.Dense(10))`
`    net.add(nn.Dense(1))`
`    net.initialize(init=init.Xavier())`
`    return net`

`k = 5`
`num_epochs = 50`
`verbose_epoch = num_epochs - 3`
`lr = 0.01`
`weight_decay = 500`
`batch_size = 64`

loss = 0.11880
### 万昕
`def get_net():`
`    net = nn.Sequential()`
`    net.add(nn.Dense(1024, activation='relu'))`
`    net.add(nn.Dropout(0.5))`
`    net.add(nn.Dense(1))`
`    net.initialize(init=init.Xavier())`
`    return net`

`k = 5`
`num_epochs = 100`
`verbose_epoch = num_epochs - 3`
`lr = 0.01`
`weight_decay = 200`
`batch_size = 64`

loss = 0.11623
